business_requirement_sale/wizard/business_requirement_create_sale.py

```diff
@@ -136,7 +131,16 @@ def _create_sale_order(self):
         self.ensure_one()
         vals = self._prepare_sale_order_vals()
         for deliverable in self.deliverable_ids:
```

Comment:
You must group deliverables by section first, and then create the section and append it, and then add its lines, all with a counter for the sequence.

Reply:
Is it totally required to have a counter and the right sequence? I tested and the lines are in order as far I could check

Reply:
What is the `_order` of the deliverable model? If it includes `section_id`, then they will be ordered when accessing `self.deliverable_ids`. If not, you need to sort them. But the sequence is needed for ensuring that no order criteria alters your order.